README.md

```diff
@@ -1,6 +1,11 @@
-# flutter-clean-architecture
+## Introduction
+
+This [VS Code](https://code.visualstudio.com/) extensions will make following clean architecture guidelines easier when using Flutter for development.
```

Comment:
Change `extensions` to `extension`.

Reply:
Change Clean Architecture to uppercase in first character.

Reply:
Done, suggested changes resolved